pug-src/new-products.pug

```diff
@@ -77,9 +77,9 @@ block content
 				li
 					| Our new BCU provides all the advantages of wireless remote
 					| control and more.
-			a.btn.btn-secondary.products(href='new-products.html' type='button')
+			a.btn.btn-secondary.products(href=baseUrl + 'new-products.html' type='button')
```

Comment:
Note: After this PR, any relative URLs will need to be done like this `baseUrl + 'path/to/file'`